tests/platforms/kubernetes/appmanager.go

```diff
@@ -43,6 +45,28 @@ type AppManager struct {
 	client    *KubeClient
 	namespace string
 	app       AppDescription
+
+	// stopChannel is the channel used to manage the port forward lifecycle
+	stopChannel chan struct{}
+	// readyChannel communicates when the tunnel is ready to receive traffic
+	readyChannel chan struct{}
+}
+
+type PortForwardRequest struct {
+	// restConfig is the kubernetes config
```

Comment:
I see comments on the side of the struct definition in the codebase.

Reply:
> I see comments on the side of the struct definition in the codebase.

I am not sure about that as I see the comments this way as well. Please refer const declaration in the same file.